### PR TITLE
Added JSON&TAP reporter support

### PR DESCRIPTION
**Pull request by m1ome**
Implemented roadmap reporters support.

### CODE REVIEW

File: src/Reporter/Json.php [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+        foreach ($results['specs'] as $type => $reports) {
+            foreach ($reports as $report) {
+                if ($report->type() !== 'pass' && $report->type() !== 'skip') {
+                    $this->_report($report);
+
+                    // Saving error report into JSON structure
+                    switch($report->type()) {
+                    	case "fail" : 
+							$this->_json["errors"][] = [
```
Comment (jails) [inline src/Reporter/Json.php]: there's a formatting issue here imo

File: src/Reporter/Tap.php [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+        $counter = 0;
+        foreach ($results['specs'] as $type => $reports) {
+            foreach ($reports as $report) {
+            	$counter++;
+
+				switch($report->type()) {
+		            case "pass":
+		            case "skip":
+					case "incomplete":
```
Comment (jails) [inline src/Reporter/Tap.php]: Same here

**Comment by jails** [comment]
:+1: !
Well done !
I'll merge it when I'll have some time to review it correctly.

**Comment by m1ome** [comment]
I will rewrite it to more consistent look.

**Pull request closed without merging**